services/apps/alcs/src/alcs/admin/unarchive-card/unarchive-card.service.ts

```diff
@@ -36,6 +40,7 @@ export class UnarchiveCardService {
     await this.fetchAndMapPlanningReviews(fileId, result);
     await this.fetchAndMapModifications(fileId, result);
     await this.fetchAndMapCovenants(fileId, result);
+    await this.fetcchAndMapNOIs(fileId, result);
```

Comment:
Typo fetcch

(Insert Mean Girls meme: Stop trying to make fetch happen)

Reply:
Thanks for the meme <3, fixed